istioctl/pkg/install/verify.go

```diff
@@ -121,7 +218,7 @@ func verifyPostInstall(enableVerbose bool, istioNamespaceFlag string,
 			for _, c := range job.Status.Conditions {
 				if c.Type == v1batch.JobFailed {
 					msg := fmt.Sprintf("Istio installation failed, incomplete or"+
-						" does not match \"%s\" - the required Job  %s failed", options.Filenames[0], name)
+						" does not match \"%s\" - the required Job  %s failed", compareFile, name)
```

Comment:
Nit: use %q instead of `\"%s\"` (several places)

Reply:
fixed

Reply:
I still see `\"%s\"` not `%q`

Reply:
there is no \"%s\" any more. please double check :)